views/admin.js

```diff
@@ -0,0 +1,20 @@
+const navigationToHomeFun = () => {
+  window.location = "/";
+};
+//Loading the current_date to date input :
+const dateDefaultValue = (document.getElementById("dateInputId").value =
```

Comment:
this should go inside the page "load" function to ensure that its called only after page is loaded and element is available